packages/fake-plugin/lib/plugin.js

```diff
@@ -45,11 +71,25 @@ class FakePlugin extends BasePlugin {
     });
   }
 
+  /**
+   *
+   * @param {Function} next - The function to be executed after this one
```

Comment:
any method in a `Plugin` meant to override a method in a `Driver` is actually a `PluginCommand` from `@appium/types`, which has the first two parameters defined for you.  `PluginCommand` looks like this:

```ts
export type PluginCommand<
  D extends ExternalDriver = ExternalDriver,
  TArgs extends readonly any[] = any[],
  TReturn = any
> = (next: NextPluginCallback, driver: D, ...args: TArgs) => Promise<TReturn>;
```

You could use `@type {PluginCommand}` here, since a) we don't care about the driver, and b) the args _really are_ `any`.